packages/super-components/components/msc-billing-tile/vue/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@ovhcloud/msc-vue-billing-tile",
+  "version": "1.0.0",
+  "description": "Vue specific wrapper for Manager super components",
```

Comment:
```suggestion
  "description": "Vue specific wrapper for Billing Tile super component",
```

Reply:
Fixed on this branch : https://github.com/ovh/manager/pull/9986